src/app/routes/edges/routes/PrivateEdgeList/index.js

```diff
@@ -1314,37 +1341,45 @@ class PrivateEdgeList extends React.Component {
                             </div>
                           </TableCell>
                           <TableCell data={n}>
-                            {this.state.userRole !== "READ_ONLY_OPS" && (
-                              <Tooltip title="Kubectl Settings">
-                                <IconButton
-                                  aria-label="edit"
-                                  className="m-0"
-                                  onClick={(event) => {
-                                    this.handleKubectlSettings(null, n);
+                            <div className="row">
+                            { isClusterReady(n) && ( 
+                                <KubeCtlShellAccess
```

Comment:
Seem like `KubeCtlShellAccess` function render `>_ KUBECTL` to the UI. If yes, should we just show icon so it is consistent with other actions like "Delete" and "Setting"? Add "Web kubectl" as tooltip to this icon?

Reply:
yeah makes sense, modified accordingly.